examples/guided.py

```diff
@@ -205,21 +269,35 @@ def ask_user_questions():
 		archinstall.arguments['kernels'] = archinstall.select_kernel(kernels)
 
 	# Additional packages (with some light weight error handling for invalid package names)
-	print("Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed.")
-	print("If you desire a web browser, such as firefox or chromium, you may specify it in the following prompt.")
+	print(
+		"Only packages such as base, base-devel, linux, linux-firmware, efibootmgr and optional profile packages are installed."
+	)
+	print(
+		"If you desire a web browser, such as firefox or chromium, you may specify it in the following prompt."
+	)
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.